modules/apps/digital-signature/digital-signature-test/src/testFunctional/tests/DigitalSignatureEnvelopeSettings.testcase

```diff
@@ -125,166 +119,96 @@ definition {
+				dmDocumentFile = "Document_2.mp4");
 		}
 
 		task ("Then the Envelope cannot be sent and an error should display indicating the file type is unsupported") {
-			AssertElementPresent(locator1 = "DigitalSignatureEnvelopeSettings#DIGITAL_SIGNATURE_UNSUPPORTED_FILE");
-
-			AssertElementPresent(
-				key_value = "Send",
-				locator1 = "DigitalSignatureEnvelopeSettings#ANY_DISABLED_BUTTON");
-		}
-	}
-
-	@description = "This is a test for LPS-131425. Verify if Success Message appear"
-	@priority = 5
-	test CanViewSuccessMessage {
-		property solutions.acceptance = "true";
+			DigitalSignature.unsupportedFile(disabledButton = "Send");
 
-		DigitalSignature.enableDigitalSignature(siteStrategy = "Always Inherit");
+			DigitalSignature.deleteCreatedDM();
 
-		DigitalSignature.createEnvelope(
-			dmDocumentFile = "Alice's Adventures in Wonderland.pdf",
-			emailMessage = "Email Message",
-			emailSubject = "Email Subject",
-			envelopeName = "Poshi Test CanViewSuccessMessage",
-			recipientEmail = "test@liferay.com",
-			recipientFullName = "Recipient Full Name");
+			RecycleBin.openRecycleBinAdmin(siteURLKey = "guest");
 
-		Button.click(button = "Send");
-
-		Alert.viewSuccessMessageText(successMessage = "Your envelope was created successfully.");
-
-		Refresh();
-
-		DigitalSignature.deleteEnvelope(envelopeName = "Poshi Test CanViewSuccessMessage");
-
-		DMNavigator.openDocumentsAndMediaAdmin(siteURLKey = "guest");
-
-		DMDocument.deleteCP(dmDocumentTitle = "DM Document Title");
-
-		RecycleBin.openRecycleBinAdmin(siteURLKey = "guest");
-
-		RecycleBin.emptyCP();
-	}
-
-	@description = "This is a test for LPS-131425. Verify if Email Subject field is required"
-	@priority = 4
-	test EmailSubjectIsRequired {
-		DigitalSignature.enableDigitalSignature(siteStrategy = "Always Inherit");
-
-		Navigator.openURL();
-
-		ProductMenu.gotoPortlet(
-			category = "Content & Data",
-			portlet = "Digital Signature");
-
-		LexiconEntry.gotoAdd();
-
-		AssertElementPresent(
-			fieldLabel = "Email Subject",
-			locator1 = "FormViewBuilder#FIELD_LABEL_REQUIRED");
+			RecycleBin.emptyCP();
+		}
 	}
 
-	@description = "This is a test for LPS-131425. Verify if Envelope Name field is required"
+	@description = "LPS-131425. verify if all required fields are been displayed"
 	@priority = 4
-	test EnvelopeNameIsRequired {
-		DigitalSignature.enableDigitalSignature(siteStrategy = "Always Inherit");
-
-		Navigator.openURL();
+	test CanSeeAllRequiredFields {
+		task ("When a User Test Goes to Digital Signature") {
+			Navigator.openURL();
 
-		ProductMenu.gotoPortlet(
-			category = "Content & Data",
-			portlet = "Digital Signature");
+			ProductMenu.gotoPortlet(
+				category = "Content & Data",
+				portlet = "Digital Signature");
 
-		LexiconEntry.gotoAdd();
+			LexiconEntry.gotoAdd();
+		}
 
-		AssertElementPresent(
-			fieldLabel = "Envelope Name",
-			locator1 = "FormViewBuilder#FIELD_LABEL_REQUIRED");
+		task ("Then it must have the Recipient Full Name field") {
```

Comment:
Please change this macro to: `"Then All required fields are being displayed"`

Reply:
Task